lockfile/src/python.rs

```diff
@@ -34,11 +32,19 @@ impl Parse for PyRequirements {
     }
 
     fn is_path_lockfile(&self, path: &Path) -> bool {
-        path.file_name() == Some(OsStr::new("requirements.txt"))
+        let file_name = match path.file_name().and_then(|f| f.to_str()) {
+            Some(f) => f,
+            None => return false,
+        };
+        file_name.starts_with("requirements") && file_name.ends_with(".txt")
```

Comment:
I feel like this would be cleaner without a match, but it's not the end of the world either.

Reply:
I guess this is equivalent to `.map(|f| f.starts_with(...) ...).unwrap_or(false)`. I can do that.

Reply:
Not sure about the exact code you plan on writing, but since you're already using a `map` into an `unwrap_or` as an example here I'd like to recommend `.map_or` instead.